CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ It diverges in the following ways:
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html) for bug fix releases.
 
 ## Unreleased
+- Removed Pocket from Homepage.
```

Comment:
I noticed we invoke a `TelemetryPocketEventPingBuilder` (defined in a-c): should we remove that too? https://searchfox.org/mozilla-mobile/search?q=telemetrypocket&path=%5Efirefox-tv&case=false&regexp=false Or would that interefere with the deletion request?

Reply:
I can create a follow-up for this in a-c.

Reply:
https://github.com/mozilla-mobile/android-components/issues/7539

Reply:
I'm concerned that we're still invoking it in FFTV: https://searchfox.org/mozilla-mobile/rev/eaea1016e1168472275c2f38b1896ff0e8722f95/firefox-tv/app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryFactory.kt#63

It seems mostly harmless - just more data transmitted (though that's more server requests and we want to avoid those) – so we _could_ leave it in. I don't know what it actually does though – maybe removing it would be harmful. Unfortunately, I don't have the brain power to figure out whether we need to remove it or not so I'll leave it up to your judgement.

Reply:
I think it makes sense to remove this.